book_scraper/book_scraper/spiders/book_spider.py

```diff
@@ -0,0 +1,51 @@
+import scrapy
+from scrapy.http import Response, Request
+from typing import Generator
+
+
+class BookSpyder(scrapy.Spider):
```

Comment:
There is a typo in the class name `BookSpyder`. It should be `BookSpider` to match the typical naming convention for spiders in Scrapy.